techniques/systemSettings/remoteAccess/sshKeyDistribution/3.0/metadata.xml

```diff
@@ -14,13 +14,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<!--
-    This is the SSH Key distribution Technique.
-    Compatibility : Linux Red Hat like, Debian like
-
-It is intended to check if all the required SSH keys are installed.
--->
-<TECHNIQUE name="SSH keys distribution">
+<TECHNIQUE name="SSH keys">
```

Comment:
I think this would be clearer if it's "SSH authorised keys" or "SSH user authorised keys" or "SSH authorised keys (users)"